docs/MIGRATION_V1_V2.md

```diff
@@ -0,0 +1,80 @@
+# Notice migration from v1.x to v2.0+
+
+The notice ID naming conventions changed in `v2` to make contributions of new rules easier by reducing the likelihood of conflicting IDs during parallel development. This table associates notices from `v1.x` and their equivalent in v2. Please refer to [RULES.md](https://github.com/MobilityData/gtfs-validator/blob/v2.0/RULES.md) to have the detailed definition of notices introduced by `v2`.
+
+| Notice ID (v1.x)                                                                 	| Notice ID (v2.0+)                                                                                                                                                                                                                                                                                                                                                                                                 	| Category (v2.0+)                                                                     	|
```

Comment:
Also, given this is a migration document now, I'd change the labels for the columns to be "v1 Notice ID", "v2 Notice Name", and "v2 Notice category".

I guess it's debatable if we should replace the 2nd column with "notice code" instead of notice name, since they are the closest things that map to each other. But IMHO it's ok to just list the name as that's more readable and can hyperlink directly to class.